ui/src/components/SchemaForm/Templates/ArrayFieldTemplate.tsx

```
@@ -0,0 +1,103 @@
+            registry={registry}
+          />
+          <div key={`array-item-list-${idSchema.$id}`}>
+            {items &&
+              items.map(
+                ({
+                  key,
+                  ...itemProps
+                }: ArrayFieldTemplateItemType<T, S, F>) => (
+                  <ArrayFieldItemTemplate key={key} {...itemProps} />
+                ),
+              )}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)